driver/src/main/scala/org/mongodb/scala/AggregateObservable.scala

```diff
@@ -164,7 +163,15 @@ case class AggregateObservable[TResult](private val wrapped: AggregateIterable[T
    * [[http://docs.mongodb.org/manual/aggregation/ Aggregation]]
    * @return a Observable with a single element indicating when the operation has completed
    */
-  def toCollection(): Observable[Completed] = observeCompleted(wrapped.toCollection(_: SingleResultCallback[Void]))
+  def toCollection(): SingleObservable[Completed] = wrapped.toCollection()
+
+  /**
+   * Helper to return a single observable limited to the first result.
+   *
+   * @return a single observable which will the first result.
+   * @since 4.0
+   */
+  def first(): SingleObservable[TResult] = wrapped.first()
```

Comment:
Looks like lots of Observables didn't have a `first()` method, probably because the scala extension to `Observable` added a `head` method.  Added to mirror the reactive streams api.